app/controllers/characters_controller.rb

```diff
@@ -0,0 +1,25 @@
+class CharactersController < ApplicationController
+  def edit
+    @character = Character.find(params[:id])
```

Comment:
I can edit & update any character, what about using:

```
@character = current_user.character.find(params[:id])
```

And hook a `before_action` to protect the action and prevent unsigned users to access the controller